Makefile

```diff
@@ -13,8 +13,8 @@ RELEASE_NAME = gohdfs-$(TRAVIS_TAG)-$(ARCH)
 all: hdfs
 
 %.pb.go: $(HADOOP_HDFS_PROTOS) $(HADOOP_COMMON_PROTOS)
-	protoc --go_out='$(PROTO_MAPPING):internal/protocol/hadoop_common' -Iinternal/protocol/hadoop_common -Iinternal/protocol/hadoop_hdfs $(HADOOP_COMMON_PROTOS)
-	protoc --go_out='$(PROTO_MAPPING):internal/protocol/hadoop_hdfs' -Iinternal/protocol/hadoop_common -Iinternal/protocol/hadoop_hdfs $(HADOOP_HDFS_PROTOS)
+	for f in $(HADOOP_COMMON_PROTOS); do protoc --go_out='$(PROTO_MAPPING):internal/protocol/hadoop_common' -Iinternal/protocol/hadoop_common -Iinternal/protocol/hadoop_hdfs $$f; done
```

Comment:
Why is this change necessary? Can you leave out any proto changes from this PR, or at least make them a separate PR?

Reply:
Also, it's weird that `make` is trying to regenerate the protos for you. They should be up-to-date in master.

Try this: 
```
find -name '*.pb.go' -exec touch {} \; && make
```

Reply:
> Maybe check the plugin has also been updated?

Yes, thanx.